projects/step-frontend/src/lib/modules/grid/components/quota-manager/quota-manager.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit, inject } from '@angular/core';
+import { downgradeComponent, getAngularJSGlobal } from '@angular/upgrade/static';
+import { AJS_MODULE } from '@exense/step-core';
+import { AugmentedQuotaManagerService } from '../../services/augmented-quota-manager.service';
+
+@Component({
+  selector: 'step-quota-manager',
+  templateUrl: './quota-manager.component.html',
+  styleUrls: ['./quota-manager.component.scss'],
+})
+export class QuotaManagerComponent implements OnInit {
+  private _augmentedQuotaManagerService = inject(AugmentedQuotaManagerService);
+  protected statusTitle!: string;
+  protected statusBody!: string;
+
+  ngOnInit(): void {
+    this.getQuotaManagerStatus();
+  }
+
+  getQuotaManagerStatus(): void {
+    this._augmentedQuotaManagerService.getQuotaManagerStatus().subscribe({
+      next: (response: string) => {
+        if (response.includes(':')) {
+          this.statusTitle = response.trim();
+          this.statusBody = '';
```

Comment:
We should do the same as for the case with "." here. Since copying would duplicate code, there should be a function:
private renderMessage(delimiter: string) : {title: string, body: string}